Makefile

```diff
@@ -145,9 +145,8 @@ endif
 ifndef HAS_DEP
 	go get -u github.com/golang/dep/cmd/dep
 endif
-ifndef HAS_GOMETALINTER
-	go get -u github.com/alecthomas/gometalinter
-	gometalinter --install
+ifndef HAS_GOLANGCI
+	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin
```

Comment:
This breaks any Windows machines from doing a `make bootstrap`.

Reply:
The alternative to wget would be to use Powershell, but I'm no expert there. Instead, maybe we can recommend [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10), I'm using that for Brigade (and Golang in general) development and works perfectly fine.

Reply:
The easiest way is to make two assumptions for local dependencies: `git` and `go` - both of which are reasonable if you are contributing to Brigade.

Then, we can clone the linter at the desired version, then go install it - or directly `go get` it, although not recommended, as it always pulls `master`.

Reply:
This way, it works across any operating system, assuming you're trying to contribute to the project, and have `git` and `go`.

Reply:
@radu-matei good call... how does the updated download procedure look?  (Pulled from https://github.com/golangci/golangci-lint#local-installation)